docs-app/README.md

```diff
@@ -0,0 +1,56 @@
+# docs-app
+
+This README outlines the details of collaborating on this Ember application.
+A short introduction of this app could easily go here.
+
+## Prerequisites
+
+You will need the following things properly installed on your computer.
+
+- [Git](https://git-scm.com/)
+- [Node.js](https://nodejs.org/) (with npm)
+- [Ember CLI](https://cli.emberjs.com/release/)
+- [Google Chrome](https://google.com/chrome/)
+
+## Installation
+
+- `git clone <repository-url>` this repository
+- `cd docs-app`
+- `npm install`
+
+## Running / Development
+
+- `ember serve`
```

Comment:
Question...are we really running `ember serve` or is it `pnpm start`? Or should updating this `README` be in a followup?

Reply:
they are equiv 🎉

Reply:
This was copy-paste, and yes, currently there is no difference. However, should we start to add turbo also to scripts in the workspace, including `start`, then it would start to matter!